src/compass/s1_cslc_qa.py

```diff
@@ -279,32 +280,134 @@ def valid_pixel_percentages(self, cslc_h5py_root):
+                                      '["never", "BasedOnNoiseMeas","always"]')),
+                                 Meta('rfi_mitigation_domain',
+                                      burst.burst_rfi_info.rfi_mitigation_domain,
+                                      'Domain the RFI mitigation step was performed')]
+                rfi_qa_items_pol += rfi_info_list
+
+            # create HDF5 group for RFI info
+            h5_rfi_path = f'{QA_PATH}/rfi_information/{burst.polarization}'
+            rfi_group = cslc_h5py_root.require_group(h5_rfi_path)
+
+            # write items to HDF5 and dict
+            _qa_items_to_h5_and_dict(rfi_group, self.rfi_dict, rfi_qa_items_pol)
+
+            # Take care of the burst RFI report information
+            if not is_rfi_info_available:
+                return
+
+            # Alias for readability
+            rfi_burst_report = burst.burst_rfi_info.rfi_burst_report
+
+            # Add the metadate of the burst RFI report
+            rfi_burst_report_list = [
+                Meta('swath',
+                     rfi_burst_report['swath'],
+                     'Swath of the burst'),
+                     Meta('azimuth_time',
+                          datetime.datetime.strftime(rfi_burst_report['azimuthTime'],
+                                                     TIME_STR_FMT),
+                          'Azimuth time of the burst report'),
+                     Meta('in_band_out_band_power_ratio',
+                          rfi_burst_report['inBandOutBandPowerRatio'],
+                          'Ratio between the in-band and out-of-band power of the burst')
+            ]
+
+            self.rfi_dict['rfi_burst_report'] = {}
+            rfi_burst_report_group = rfi_group.require_group('rfi_burst_report')
+            _qa_items_to_h5_and_dict(rfi_burst_report_group,
+                                     self.rfi_dict['rfi_burst_report'],
+                                     rfi_burst_report_list)
+
+            # Take care of the time domain portion of the burst report
+            if 'timeDomainRfiReport' in rfi_burst_report.keys():
+                time_domain_report = rfi_burst_report['timeDomainRfiReport']
+                burst_time_domain_report_item = [
+                    Meta('percentage_affected_lines',
+                         time_domain_report['percentageAffectedLines'],
+                         'Percentage of level-0 lines affected by RFI.'),
+                    Meta('avg_percentage_affected_samples',
+                         time_domain_report['avgPercentageAffectedSamples'],
+                         'Average percentage of affected level-0 samples in the lines containing RFI'),
+                    Meta('max_percentage_affected_samples',
+                         time_domain_report['maxPercentageAffectedSamples'],
+                         'Maximum percentage of level-0 samples affected by RFI in the same line'),
+                ]
```

Comment:
```suggestion
                ]

```
nit - Add a line for consistency with all the other metadata add blocks.